### ISSUE TITLE
about loss function

### ISSUE THREAD

**Issue by Yisher** [GuoxunZhang-THU/DeepSeMi#2]
Hello !Great work.
I don't understand the definition of the loss function beacause I don't see a detailed description of the loss function in the paper，Is the loss function the L1 and L2 loss between the output video through FFnet and the original input video? Part of the confusion is about self-supervision, I want to know what the real_A, fake_A, and real_B refer to in the loss function code,and what does 'B2A' mean ; Another part of the puzzle is why the loss function is designed in the form of a combination of L1 and L2. Thank you very much for your work and clarification.
`self.optimizer.zero_grad()
                # L1Loss_A2B = L1loss_function(train_imB, pred_imA)
                L1Loss_B2A = self.L1loss_function(real_B, fake_A)
                L2Loss_B2A = self.L2loss_function(real_B, fake_A)
                loss = L2Loss_B2A+L1Loss_B2A

                real_B_save = real_B[ :, :, int(real_B.shape[2]*0.25):int(real_B.shape[2]*0.75), int(real_B.shape[3]*0.25):int(real_B.shape[3]*0.75), int(real_B.shape[4]*0.25):int(real_B.shape[4]*0.75), ]
                fake_A_save = fake_A[ :, :, int(fake_A.shape[2]*0.25):int(fake_A.shape[2]*0.75), int(fake_A.shape[3]*0.25):int(fake_A.shape[3]*0.75), int(fake_A.shape[4]*0.25):int(fake_A.shape[4]*0.75), ]
                loss_save = self.L1loss_function(real_B_save, fake_A_save) + self.L2loss_function(real_B_save, fake_A_save)
                loss_m.append(loss_save.cpu().detach().numpy())

                for sub_x_s_i in range(len(sub_x_s_list)):
                    sub_x_s = sub_x_s_list[sub_x_s_i]
                    loss = loss + self.L1loss_function(real_B, sub_x_s) + self.L2loss_function(real_B, sub_x_s)

                for sub_x_t_i in range(len(sub_x_t_list)):
                    sub_x_t = sub_x_t_list[sub_x_t_i]
                    loss = loss + self.L1loss_function(real_B, sub_x_t) + self.L2loss_function(real_B, sub_x_t)

                loss.backward()
                self.optimizer.step()`
